processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/ProcessingChainBDTopoTest.groovy

```diff
@@ -58,10 +52,39 @@ class ProcessingChainBDTopoTest extends ChainProcessAbstractTest{
         }
     }
 
+    def loadFiles(String inseeCode){
+        H2GIS h2GISDatabase = H2GIS.open(h2db, "sa", "")
+        String relativePath = bdtopoFoldName + File.separator + inseeCode
+
+        // Test whether there is a folder containing .shp files for the corresponding INSEE code
+        if(ProcessingChain.class.getResource(relativePath) != null){
+            // Test is the URL is a folder
+            if(new File(ProcessingChain.class.getResource(relativePath).toURI()).isDirectory()){
+                listTables.each {
+                    String filePath = ProcessingChain.class.getResource(relativePath + File.separator + it + ".shp")
```

Comment:
It might fail because `getResource()` return an `URL` and you are forcing Groovy to cast it as `String`. You should use `def filePath = ...`.

Reply:
done